src/components/Header/Header.tsx

```diff
@@ -0,0 +1,53 @@
+import classNames from 'classnames';
+
+import React from 'react';
+
+type Props = {
+  title: string,
+  setTitle: (value: string) => void,
+  onSubmit: (event: React.FormEvent) => void,
+  isDisabled: boolean,
+  toggleAll: () => void,
+  noCompletedTodos: boolean,
+};
+
+export const Header: React.FC<Props> = React.memo(
+  ({
+    title,
+    setTitle,
+    onSubmit,
+    isDisabled,
+    toggleAll,
+    noCompletedTodos,
+  }) => {
+    return (
+      <header className="todoapp__header">
+        {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
+        <button
+          type="button"
+          className={classNames(
+            'todoapp__toggle-all',
+            { active: noCompletedTodos },
+          )}
+          onClick={toggleAll}
+        />
```

Comment:
```suggestion
     
        <button
          type="button"
          className={classNames(
            'todoapp__toggle-all',
            { active: noCompletedTodos },
          )}
          onClick={toggleAll}
        ></button>
```

Reply:
I tried to make it a lot of times, but when I run linter, is automatically back to `<button/>`